source/sensdet/ChargeHit.h

```diff
@@ -0,0 +1,76 @@
+// ----------------------------------------------------------------------------
```

Comment:
Don't we need a `ChargeHit.cc` file as well?

Reply:
Why?

Reply:
Because it is the source file and the rest of classes of the directory have it.

Reply:
If the functions are simple enough, there is no need to create a `.cc` file. I think this is the case.